shepherd/Utils.py

```diff
@@ -1,58 +1,99 @@
 # pylint: disable=invalid-name
 class SHEPHERD_HEADER():
```

Comment:
Why aren't these enums again? [You can use strings for enums](https://docs.python.org/3/library/enum.html#others).

If these _must_ be regular classes, at least get rid of the meaningless `()` (already inherits from `object` without the braces).

Rename to `ShepherdHeader` to conform with Python class naming convention.